### PR TITLE
Update the release captain howto to the latest workflow.

### PR DESCRIPTION
**Pull request by votdev**
Signed-off-by: Volker Theile <vtheile@suse.com>

# Describe your changes

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.
- [ ] CHANGELOG.md has been updated should there be relevant changes in this PR.

### CODE REVIEW

**Comment by votdev** [comment]
I'm not happy with the changelog processing. It's much too complicated right now. Updating the release date right before the release requires now 2 PRs per sub-project:

1. Do the changes in `main`.
2. Backport them to the release branch.

Doing that for `s3gw-ui`, `s3gw-tools` and `s3gw-charts` requires at least 6 PRs. That's too much. Getting them reviewed, accepted and merged will surely slow down the release process dramatically.

**Comment by jhmarina** [comment]
> 

[There's this issue](https://github.com/aquarist-labs/s3gw/issues/166) for merging the tools & charts repo to simplify the process a bit

**Comment by votdev** [comment]
> > 
> 
> [There's this issue](https://github.com/aquarist-labs/s3gw/issues/166) for merging the tools & charts repo to simplify the process a bit

That will simplify the release process, but IMO it is to late to include this into `0.8.0`. The next release captain will need to update the ADR after #166 has been merged.

**Review by jhmarina — COMMENTED** [review]
lgtm

**Comment by giubacc** [comment]
it seems good; remind me, what was the need for an annotated tag?

**Comment by votdev** [comment]
> it seems good; remind me, what was the need for an annotated tag?

Check https://git-scm.com/book/en/v2/Git-Basics-Tagging. The need is that only this type of tag triggers the GH workflow to build the images. @m-ildefons can give more details on that.

**Review by giubacc — DISMISSED** [review]

**Comment by m-ildefons** [comment]
It needs to be annotated (`-a` flag) and IMO it needs to be signed (`-s` flag) too, which afaik can't be done with the web interface. The command line for that is:
```
git tag -a -s vX.Y.Z
```

File: docs/decisions/0004-release-captain.md
Comment (jecluis) [inline docs/decisions/0004-release-captain.md]: needs to be done for `ceph.git` as well.

**Review by jecluis — CHANGES_REQUESTED** [review]

File: docs/decisions/0004-release-captain.md
Comment (votdev) [reply docs/decisions/0004-release-captain.md]: Done

File: docs/decisions/0004-release-captain.md
Comment (jecluis) [inline docs/decisions/0004-release-captain.md:40]: sorry, should have mentioned this before. Might be relevant to say that the changelog file under `ceph` is in `src/rgw/store/sfs/`. Just so we don't update the wrong file.

**Review by jecluis — CHANGES_REQUESTED** [review]

File: docs/decisions/0004-release-captain.md
Comment (votdev) [reply docs/decisions/0004-release-captain.md:40]: Done

**Review by jecluis — APPROVED** [review]

**Pull request merged**
